test/functional/api/cas/cli_messages.py

```diff
@@ -15,6 +15,14 @@
     r"Successfully added cache instance \d+"
 ]
 
+start_wo_load_on_cache_dev_prev_used = [
```

Comment:
rename to sth more meaningful like start_cache_with_existing_metadata
BTW 'wo' can be misunderstood, especially that we have WO mode

Reply:
good point